tech/languages/csharp/dotnet-installation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: C# - .NET Core
+subsection: csharp
+section: tech-languages
+order: 2
+version: 7.8.4
+description: "C# programming language, .NET Core and Mono open source frameworks."
+---
+
+# .NET SDK installation
+
+.NET Core is still work in progress, and below download locations contain packages that are not entirely _free._ Please be patient, we should have reasonable packages within a few months.
+
+Choose based on your Fedora and SDK version:
+
+* [Microsoft's 2.0 preview1](https://www.microsoft.com/net/core/preview#linuxfedora) (F25 `.csproj`)
+* [Microsoft's 1.X packages](https://www.microsoft.com/net/download/linux) (F23 & 24 `.json`)
+* [Fedora .NET SIG packages](https://copr.fedorainfracloud.org/coprs/nmilosev/dotnet-sig) (F25+ `.json`)
```

Comment:
This is due to Microsoft releasing anything and everything a year late. e.g. they released the official 1.x package for F23 a month before it was eol... (iirc)

I removed F23 reference, we will replace all of these weird packages once we get ours out - working on the build now, we should have `.NET Core 2.0 Preview2` released within a week or two - for 25 and 26 (hopefully we won't hit any major issues on 26...)